Gulpfile.js

```diff
@@ -379,12 +379,6 @@ gulp.step('test-functional-local-run', () => {
 
 gulp.task('test-functional-local', gulp.series('prepare-tests', 'test-functional-local-run'));
 
-gulp.step('test-functional-local-ie-run', () => {
-    return testFunctional(TESTS_GLOB, functionalTestConfig.testingEnvironmentNames.localBrowsersIE);
```

Comment:
Also, please remove the `functionalTestConfig.testingEnvironmentNames.localBrowsersIE` field from the `test/functional/config.js` file.